tests/Wallabag/CoreBundle/CoreKernelTestCase.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace tests\Wallabag\CoreBundle;
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+abstract class CoreKernelTestCase extends KernelTestCase
+{
+    protected $fetchingErrorMessage;
+
+    public function setUp()
+    {
+        self::bootKernel();
+        $container = self::$kernel->getContainer();
+        $this->fetchingErrorMessage = $container->getParameter('wallabag_core.fetching_error_message');
```

Comment:
I'm against that. Sorry @nicosomb 

Because you boot the kernel when you don't necessary need it. And most case where you're going to use that message doesn't need the Symfony app to be booted.
You should better use a plain text when you need the fetching error message.

Taking example from `ContentProxyTest.php`:

```php
class ContentProxyTest extends \PHPUnit_Framework_TestCase
{
    private $fetchingErrorMessage = 'We can process it';

    public function testWithBadUrl()
    {
        // ...
        $proxy = new ContentProxy($graby, $tagger, $this->getTagRepositoryMock(), $this->getLogger(), $this->fetchingErrorMessage);
    }
}
```